client/extensions/woocommerce/components/order-status/style.scss

```
@@ -35,11 +35,11 @@
 	}
 
 	&.is-on-hold {
-		background: lighten( $alert-yellow, 20% );
-		color: darken( $alert-yellow, 40% );
+		background: var( --color-warning-100 );
```

```suggestion
		background: var( --color-warning-50 );
```
Let's do 50 to lighten the backgroutd a bit. Otherwise, LGTM